.circleci/config.yml

```diff
@@ -39,6 +39,12 @@ jobs:
     working_directory: ~/node-10
     steps: *buildSteps
 
+  "node-12":
+    docker:
+      - image: circleci/node:12
+    working_directory: ~/node-12
+    steps: *buildSteps
```

Comment:
Cameron, if you are an admin of this repo, you might be able to replace Node 6 with Node 12 as a required test.